tests/safeds/data/tabular/transformation/test_discretizer.py

```diff
@@ -11,14 +11,34 @@ def test_should_raise_value_error(self) -> None:
 
 
 class TestFit:
-    def test_should_raise_if_column_not_found(self) -> None:
-        table = Table(
-            {
-                "col1": [0.0, 5.0, 10.0],
-            },
-        )
-
-        with pytest.raises(UnknownColumnNameError):
+    @pytest.mark.parametrize(
+        ("table", "error", "error_message"),
+        [
+            (
+                Table(
+                    {
+                        "col1": [0.0, 5.0, 5.0, 10.0],
+                    },
+                ),
+                UnknownColumnNameError,
+                r"Could not find column\(s\) 'col2'",
+            ),
```

Comment:
Please add a test that tests if the correct error message is raised when there are multiple missing columns. See #407 for how you can sort the column names.